src/cascade/dismod/db/wrapper.py

```diff
@@ -187,15 +181,40 @@ def __dir__(self):
         attributes.extend(super().__dir__())
         return attributes
 
+    def update_table_columns(self, table_definition, table):
+        new_columns = table.columns.difference(table_definition.c.keys())
+        new_columns = {c: table.dtypes[c] for c in new_columns}
```

Comment:
The values change here, so it would be easier to notice that if the variable name is different from `new_columns`.

Reply:
Well, here's a readability question. I'm doing this for the sake of not having an absurdly complex one-liner (and instead having two moderately complex one-liners). The first value of `new_columns` is totally ephemeral. Is it more clear to throw that ephemeral value away but maybe muddy peoples' understanding of what the second line does? Or is it more clear to not have the dangling value, which I'll never actually use, and make it clear that there is a transformation in the second line?

Reply:
I was confused about why this wrote twice to a variable since the first was thrown away. And Code Complete says reuse of names is a source of errors. Thus sayeth Code Complete.

Reply:
Though I continue to object to the appeal to dubious authority, if you were confused then that's meaningful. I've reorganized it a bit so it should be more clear.